app/controllers/report_controller/reports/editor.rb

```diff
@@ -343,7 +343,7 @@ def build_tabs
         ["#{req}_3", "Filter"],
         ["#{req}_7", "Preview"]
       ]
-    elsif @edit[:new][:model] == "Chargeback"
+    elsif @edit[:new][:model].to_s.starts_with?("Chargeback")
```

Comment:
Not required for this PR - These calls should be be refactored into a local variable or a method.